thirdparty/README.md

```diff
@@ -1,5 +1,11 @@
 # Third party libraries
 
+## doctest
```

Comment:
This doctest section should go between the `cvtt` and `enet` sections, as these sections are sorted alphabetically.

Reply:
Sounds good, doing that now